docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -285,8 +285,237 @@ dependencies {
 
 ### Native Code
 
+The last step requires us to write some native code to connect the JS side of our Component to what is offered by the platforms. This process requires two main steps:
+
+- Run the **CodeGen** to see what would be generated
+- Write the native code that will make it work
+
+:::caution
+The code generated by the **CodeGen** in this step should not be committed to the versioning system. React Native apps are able to generate
+the code when the app is built. This allows to avoid any ABI incompatibility and to ensure that a consistent version of the codege is used.
+:::
+
 #### iOS
 
+##### Generate the code
+
+To generate the code starting from the JS specs for iOS, we need to open a terminal and run the following command:
+
+```sh
+cd MyApp
+yarn add ../RTNCenteredTextFlow
+cd ..
+node MyApp/node_modules/react-native/scripts/generate-artifacts.js \
+  --path MyApp/ \
+  --outputPath RTNCenteredTextFlow/generated/
```

Comment:
To run the CodeGen, we need a React Native app (which can access the React Native code). A module alone does not have that dependency, therefore we need to add the module to the app.

Once that is done, we can invoke the CodeGen manually to see what it will outputs, so that we can use the right filenames and types.
By invoking the codegen manually, we need to define an output folder and I thought it was easier to use the module folder as root. 

As specified in the **caution** block above, the code generated manually should not be committed to the code versioning system and will be discarded. React Native relies on the code generated automatically when building the app.

Reply:
Cool, that makes sense -- it might be good to briefly explain the intention here, the reader may wonder (as I did) if this process is necessary in the normal course of development

Reply:
I added a couple of paragraphs before the `iOS` section to describe that. Let me know what do you think!